packages/core/src/shared/clients/ec2Wrapper.ts

```diff
@@ -17,32 +38,33 @@ import { decodeBase64 } from '../utilities/textUtilities'
 /**
  * A wrapper around EC2.Instance where we can safely assume InstanceId field exists.
  */
-export interface SafeEc2Instance extends EC2.Instance {
+export interface SafeEc2Instance extends Instance {
     InstanceId: string
     Name?: string
-    LastSeenStatus: EC2.InstanceStateName
+    LastSeenStatus: InstanceStateName
 }
 
-interface SafeEc2GetConsoleOutputResult extends EC2.GetConsoleOutputRequest {
+interface SafeEc2GetConsoleOutputResult extends GetConsoleOutputRequest {
     Output: string
     InstanceId: string
 }
 
-export class Ec2Client {
+export class Ec2Wrapper {
```

Comment:
Out of curiosity what was the idea between calling it Ec2Wrapper? Is it because it's a toolkit wrapper around the sdk v3 builder?

Reply:
Yeah it seems like a more fitting name, and also because the EC2 v3 exports its client as `EC2Client` (instead of `EC2` as in v2) so want to avoid confusion between the names. Do you think that renaming the 'clients' to 'wrappers' as we migrate them makes sense?